Tests/PointFreeTests/PrivateRssTests.swift

```diff
@@ -176,6 +181,36 @@ class PrivateRssTests: TestCase {
 
     let conn = connection(from: req)
 
+    assertSnapshot(matching: conn |> siteMiddleware, as: .ioConn)
+    XCTAssertTrue(feedRequestEventCreated)
```

Comment:
this test confirms that the `createFeedRequestEvent` database endpoint is called even in the invalid UA flow.